Aptoide/appcoins-wallet-android
# [MOB-892] fingerprint entering wallet

**What does this PR do?**

When entering the wallet, a fingerprint authentication method will be asked in order to continue the navigation. Also added an option on the settings to turn on and off this feature.

**Database changed?**

No

**Where should the reviewer start?**

- SplashActivity.java
- AuthenticationPromptActivity.kt

**How should this be manually tested?**

Open the wallet with a device with biometric credentials and check if the authentication is asked.
Open the wallet with a device that doesn't have any biometric credentials and check if the wallet navigates normally.
Open the wallet with the settings On and Off.


**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-892

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
